packages/dds/tree/src/util/brandedMap.ts

```diff
@@ -3,11 +3,16 @@
  * Licensed under the MIT License.
  */
 
+import type { Brand } from "./brand.js";
+import type { Opaque } from "./opaque.js";
 import type { Invariant } from "./typeCheck.js";
 import { getOrCreate } from "./utils.js";
 
 /**
  * Key in a {@link BrandedMapSubset}.
+ * @remarks
```

Comment:
Nit: this seems more like a framework developer-facing note. Would `@privateRemarks` make more sense here?

Reply:
Types using this type end up needing to refer to this to justify their use of any, so I think it makes sense to be remarks. This entire API surface is internal though, which makes even the remarks framework developer-facing.

Reply:
Just adding a data point: for internal APIs, since no one but us is expected to be able to see those docs, I'd interpret `@remarks` to be targeting framework developers already. Counter-argument, prevent those docs from surfacing by mistake if the API ever went public, but in that case in-depth docs review will (should) probably also happen and things should be adjusted there.